cetmix_tower_server/tests/test_cetmix_tower.py

```diff
@@ -90,3 +91,42 @@ def test_server_get_variable_value(self):
             self.server_test_1.reference, variable_meme.reference
         )
         self.assertEqual(value, server_value.value_char)
+
+    def test_server_check_ssh_connection_success(self):
+        """
+        Test the successful SSH connection check
+        """
+        with patch(
+            "odoo.addons.cetmix_tower_server.models.cx_tower_server.SSH.connection",
+            return_value=None,
```

Comment:
Function you are testing should return either a dictionary or raise an exception.
One of the options to achieve this is to create a mock function that returns pre-defined result for different input data.  And then replace real function with the mock none like it's done [here](https://github.com/cetmix/cetmix-tower/blob/41a49cb60f4d38fbcd1008c7a931519863fa7ad4/cetmix_tower_server/tests/test_plan.py#L1046).